### PR TITLE
Fix typos in properties

### PR DESCRIPTION
**Pull request by k-tamura**
I found some typos in properties. Could you check them?

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties [hunk 1]
```diff
@@ -19,7 +19,7 @@ registerWithTitleHtml={0}
 loginTitle=Inicia sessi\u00F3 a {0}
 loginTitleHtml={0}
 impersonateTitle={0}\u00A0Personifica Usuari
-impersonateTitleHtml=<strong>{0}</strong> Personifica Usuari</strong>
```
Comment (stianst) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties]: strong is already closed earlier

File: themes/src/main/resources/theme/base/account/messages/messages_en.properties [hunk 2]
```diff
@@ -122,7 +122,7 @@ invalidEmailMessage=Invalid email address.
 missingLastNameMessage=Please specify last name.
 missingEmailMessage=Please specify email.
 missingPasswordMessage=Please specify password.
-notMatchPasswordMessage=Passwords don''t match.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/account/messages/messages_en.properties]: Java properties require ' to be escaped, hence why it's '' and not just '

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 3]
```diff
@@ -91,11 +91,11 @@ realm-cache-clear.tooltip=Clears all entries from the realm cache (this will cle
 user-cache-clear=User Cache
 user-cache-clear.tooltip=Clears all entries from the user cache (this will clear entries for all realms)
 keys-cache-clear=Keys Cache
-keys-cache-clear.tooltip=Clears all entries from the cache of external public keys. These are keys of external clients or identity providers. (this wil clear entries for all realms)
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 4]
```diff
@@ -91,11 +91,11 @@ realm-cache-clear.tooltip=Clears all entries from the realm cache (this will cle
...
+keys-cache-clear.tooltip=Clears all entries from the cache of external public keys. These are keys of external clients or identity providers. (this will clear entries for all realms)
 revoke-refresh-token=Revoke Refresh Token
 revoke-refresh-token.tooltip=If enabled a refresh token can only be used up to 'Refresh Token Max Reuse' and is revoked when a different token is used. Otherwise refresh tokens are not revoked when used and can be used multiple times.
 refresh-token-max-reuse=Refresh Token Max Reuse
-refresh-token-max-reuse.tooltip=Maximum number of times a refresh token can be reused. When a different token is used, revokation is immediate.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 5]
```diff
@@ -124,7 +124,7 @@ action-token-generated-by-user.reset=Reset
 action-token-generated-by-user.operation=Override User-Initiated Action Lifespan
 
 client-login-timeout=Client login timeout
-client-login-timeout.tooltip=Max time an client has to finish the access token protocol. This should normally be 1 minute.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 6]
```diff
@@ -290,7 +290,7 @@ valid-redirect-uris=Valid Redirect URIs
 valid-redirect-uris.tooltip=Valid URI pattern a browser can redirect to after a successful login or logout. Simple wildcards are allowed i.e. 'http://example.com/*'. Relative path can be specified too i.e. /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.
 base-url.tooltip=Default URL to use when the auth server needs to redirect or link back to the client.
 admin-url=Admin URL
-admin-url.tooltip=URL to the admin interface of the client. Set this if the client supports the adapter REST API. This REST API allows the auth server to push revocation policies and other adminstrative tasks. Usually this is set to the base URL of the client.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 7]
```diff
@@ -549,8 +549,8 @@ select-account.option=select_account
 prompt.tooltip=Specifies whether the Authorization Server prompts the End-User for reauthentication and consent.
 validate-signatures=Validate Signatures
 identity-provider.validate-signatures.tooltip=Enable/disable signature validation of external IDP signatures.
-identity-provider.use-jwks-url.tooltip=If the switch is on, then identity provider public keys  will be downloaded from given JWKS URL. This allows great flexibility because new keys will be always re-downloaded again when identity provider generates new keypair. If the switch is off, then public key (or certificate) from the Keycloak DB is used, so when identity provider keypair changes, you always need to import new key to the Keycloak DB as well.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 8]
```diff
@@ -549,8 +549,8 @@ select-account.option=select_account
...
-identity-provider.jwks-url.tooltip=URL where identity provider keys in JWK format are stored. See JWK specification for more details. If you use external keycloak identity provider, then you can use URL like 'http://broker-keycloak:8180/auth/realms/test/protocol/openid-connect/certs' assuming your brokered keycloak is running on 'http://broker-keycloak:8180' and it's realm is 'test' .
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 9]
```diff
@@ -594,7 +594,7 @@ social.default-scopes.tooltip=The scopes to be sent when asking for authorizatio
 key=Key
 stackoverflow.key.tooltip=The Key obtained from Stack Overflow client registration.
 openshift.base-url=Base Url
-openshift.base-url.tooltip=Base Url to Openshift Online API
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 10]
```diff
@@ -632,7 +632,7 @@ back=Back
 initial-access-tokens=Initial Access Tokens
 add-initial-access-tokens=Add Initial Access Token
 initial-access-token=Initial Access Token
-initial-access.copyPaste.tooltip=Copy/paste the initial access token before navigating away from this page as it's not posible to retrieve later
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 11]
```diff
@@ -869,7 +869,7 @@ reset-credentials.tooltip=Select the flow you want to use when the user has forg
 client-authentication=Client Authentication
 client-authentication.tooltip=Select the flow you want to use for authentication of clients.
 docker-auth=Docker Authentication
-docker-auth.tooptip=Select the flow you want to use for authenticatoin against a docker client.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 12]
```diff
@@ -1126,7 +1126,7 @@ authz-any-scope=Any scope
 authz-any-role=Any role
 authz-policy-evaluation=Policy Evaluation
 authz-select-client=Select a client
-authz-select-user=Select an user
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 13]
```diff
@@ -1164,7 +1164,7 @@ authz-create-permission=Create Permission
 
 # Authz Resource Detail
 authz-add-resource=Add Resource
-authz-resource-name.tooltip=An unique name for this resource. The name can be used to uniquely identify a resource, useful when querying for a specific resource.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 14]
```diff
@@ -1175,7 +1175,7 @@ authz-add-scope=Add Scope
 authz-no-scopes-available=No scopes available.
 
 # Authz Scope Detail
-authz-scope-name.tooltip=An unique name for this scope. The name can be used to uniquely identify a scope, useful when querying for a specific scope.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 15]
```diff
@@ -1284,7 +1284,7 @@ authz-permission-scope-scope.tooltip=Specifies that this permission must be appl
 authz-evaluation-identity-information=Identity Information
 authz-evaluation-identity-information.tooltip=The available options to configure the identity information that will be used when evaluating policies.
 authz-evaluation-client.tooltip=Select the client making this authorization request. If not provided, authorization requests would be done based on the client you are in.
-authz-evaluation-user.tooltip=Select an user whose identity is going to be used to query permissions from the server.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 16]
```diff
@@ -1364,11 +1364,11 @@ permissions-enabled-role=Permissions Enabled
 permissions-enabled-role.tooltip=Whether or not to enable fine grain permissions for managing this role.  Disabling will delete all current permissions that have been set up.
 manage-permissions-role.tooltip=Fine grain permissions for managing roles.  For example, you can define different policies for who is allowed to map a role.
 lookup=Lookup
-manage-permissions-users.tooltip=Fine grain permssions for managing all users in realm.  You can define different policies for who is allowed to manage users in the realm.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 17]
```diff
@@ -1364,11 +1364,11 @@ permissions-enabled-role=Permissions Enabled
...
+manage-permissions-users.tooltip=Fine grain permissions for managing all users in realm.  You can define different policies for who is allowed to manage users in the realm.
 permissions-enabled-users=Permissions Enabled
 permissions-enabled-users.tooltip=Whether or not to enable fine grain permissions for managing users.  Disabling will delete all current permissions that have been set up.
-manage-permissions-client.tooltip=Fine grain permssions for admins that want to manage this client or apply roles defined by this client.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties [hunk 18]
```diff
@@ -1364,11 +1364,11 @@ permissions-enabled-role=Permissions Enabled
...
-manage-permissions-group.tooltip=Fine grain permssions for admins that want to manage this group or the members of this group.
```
Comment (stianst) [inline themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties]: OK

File: themes/src/main/resources/theme/base/login/messages/messages_en.properties [hunk 19]
```diff
@@ -233,7 +233,7 @@ staleCodeMessage=This page is no longer valid, please go back to your applicatio
 realmSupportsNoCredentialsMessage=Realm does not support any credential type.
 identityProviderNotUniqueMessage=Realm supports multiple identity providers. Could not determine which identity provider should be used to authenticate with.
 emailVerifiedMessage=Your email address has been verified.
-staleEmailVerificationLink=The link you clicked is a old stale link and is no longer valid.  Maybe you have already verified your email?
```
Comment (stianst) [inline themes/src/main/resources/theme/base/login/messages/messages_en.properties]: OK

**Review by stianst — CHANGES_REQUESTED** [review]
* strong tags are already closed earlier
* ' needs to be escaped in Java properties hence it's ''
* Added an OK next to the changes that looks good, the rest is not good

**Comment by stianst** [comment]
Please create a JIRA as well

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties (hunk 1 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties]: Yes. This is an unnecessary tag. Do you think no need to remove?

File: themes/src/main/resources/theme/base/account/messages/messages_en.properties (hunk 2 above)
Comment (k-tamura) [reply themes/src/main/resources/theme/base/account/messages/messages_en.properties]: Sorry, I have misunderstood. I will remove them.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties (hunk 1 above)
Comment (stianst) [reply themes/src/main/resources-community/theme/base/login/messages/messages_ca.properties]: The additional </strong> you've added to the end needs to be removed

**Pull request closed without merging**